pkg/test/echo/client/client.go

```diff
@@ -98,21 +99,21 @@ func (c *Instance) Close() error {
 	return nil
 }
 
-func (c *Instance) Echo(ctx context.Context, request *proto.EchoRequest) (*ParsedResponse, error) {
+func (c *Instance) Echo(ctx context.Context, request *proto.EchoRequest) (*response.Instance, error) {
```

Comment:
mentioned earlier, but might be an opportunity to stop using pointers.

Also while I am not the nitpicks, I would have a slight prefernce for this to be `echo.Response` and `echo.Responses`... not a big deal but we have a lot of "Instance"s that get confusing